members.md

```diff
@@ -24,6 +24,48 @@ If the request succeeds, `meta.errors` will be null, and if the request fails, `
 
 ***
 
+## Index Members
+
+Fetch a group's current or former member list.
+
+This call is limited to admins and owners in the group. Any other caller will receive a `401 Unauthorized` response.
+
+**Request**
+`GET /groups/:group_id/members`
+
+**Parameters**
+
+* *filter* (required)
+
+	string - to fetch either `active` (current memberships) or `inactive` (former memberships).
+	
+**Responses**
+```
+Status: 200 OK
+{
+  "memberships":[
+    {
+      "id": "24681012",
+	     "user_id": "11223344",
+	     "name": "Alureon",
+	     "nickname": "Alu",
+	     "image_url": "https://i.groupme.com/1a3c5e7g",
+     	"state": "active",
+	     "roles": ["owner","admin"]
```

Comment:
formatting again

Reply:
I can't see this in my editor... It looks fine on my end. I'm not sure what's up.